python/src/scipp/compat/mantid.py

```diff
@@ -204,35 +204,19 @@ def _to_spherical(pos, output):
     sc.atan2(sc.geometry.y(pos), sc.geometry.x(pos), output["p"].data)
 
 
-def rotation_matrix_from_vectors(vec1, vec2):
-    """ Find the rotation matrix that aligns vec1 to vec2
-    :param vec1: A 3d "source" vector
-    :param vec2: A 3d "destination" vector
-    :return mat: Tranform (3x3) which aligns it with vec2.
-    """
-    a, b = (vec1 /
-            np.linalg.norm(vec1)).reshape(3), (vec2 /
-                                               np.linalg.norm(vec2)).reshape(3)
-    v = np.cross(a, b)
-    c = np.dot(a, b)
-    s = np.linalg.norm(v)
-    if s == 0:
-        return np.eye(3)
-    kmat = np.array([[0, -v[2], v[1]], [v[2], 0, -v[0]], [-v[1], v[0], 0]])
-    rotation_matrix = np.eye(3) + kmat + kmat.dot(kmat) * ((1 - c) / (s**2))
-    return rotation_matrix
-
-
-def matrix_mult(pos, m):
-    if not pos.shape[0]:
-        return pos
-    m1 = sc.Variable(value=m[0, :], dtype=sc.dtype.vector_3_float64)
-    m2 = sc.Variable(value=m[1, :], dtype=sc.dtype.vector_3_float64)
-    m3 = sc.Variable(value=m[2, :], dtype=sc.dtype.vector_3_float64)
-    xn = sc.dot(m1, pos)
-    yn = sc.dot(m2, pos)
-    zn = sc.dot(m3, pos)
-    return sc.geometry.position(xn, yn, zn)
+def _quat_from_vectors(vec1, vec2):
```

Comment:
Maybe there is a place we could put high-level generic python utilities that are using Scipp?
Something like `scipp.tools` or `scipp.math`?
I have something like that mainly for plotting purposes, but some mathematical ones might be useful, instead of this being part of `mantid.py`?

Reply:
Agreed. I'll make a note of this as I'm going to be refactoring `mantid.py` anyway as part of the clean up.